utils/src/main/java/io/neow3j/constants/InteropServiceCode.java

```diff
@@ -45,7 +45,8 @@
 
     // Native tokens.
     NEO_NATIVE_TOKENS_NEO("Neo.Native.Tokens.NEO", null),
-    NEO_NATIVE_TOKENS_GAS("Neo.Native.Tokens.GAS", null);
+    NEO_NATIVE_TOKENS_GAS("Neo.Native.Tokens.GAS", null),
+    NEO_NATIVE_POLICY("Neo.Native.Policy", null); // TODO: 24.06.20 Michael: Is price null?
```

Comment:
hmmm.. what do you mean by "is price null"? 😄 
like, the paid gas?

Reply:
@csmuller just a clarification: so... it means that when the "gas price" is `null` in this file, there's some kind of dynamic calculation happening somewhere?

Reply:
> hmmm.. what do you mean by "is price null"? 😄
> like, the paid gas?

Yes, the execution GAS price. @csmuller just told me, it actually should be set to null 👍 , so I'll remove the todo.

> @csmuller just a clarification: so... it means that when the "gas price" is `null` in this file, there's some kind of dynamic calculation happening somewhere?

Correct.